packages/frontend/src/components/ShareLinkButton/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import { useApp } from '../../providers/AppProvider';
+import { ShareLink } from './ShareLinkButton.styles';
+
+const ShareLinktButton: FC = () => {
+    const { showToastSuccess } = useApp();
+
+    const onClick = () => {
+        const linkToShare = window.location.href;
```

Comment:
We better use `import CopyToClipboard from 'react-copy-to-clipboard';` 
You can check how we use it in `packages/frontend/src/components/UserSettingsModal/InvitesPanel/index.tsx`